api.md

```diff
@@ -654,8 +621,10 @@ SELECT set_chunk_time_interval('conditions', 86400000);
 
 ---
 
-## set_adaptive_chunking() [](set_adaptive_chunking)
-Changes the settings for [adaptive chunking (BETA)][adaptive-chunking]. The
+## DEPRECATED set_adaptive_chunking() [](set_adaptive_chunking)
+
+The adaptive chunking feature is now deprecated and should not be used.
```

Comment:
Perhaps this first line ("The adaptive chunking feature is now deprecated and should not be used.") should be a :WARNING: ?